.github/workflows/mac_unit_test_ci.yml

```diff
@@ -13,5 +13,5 @@ jobs:
     - uses: actions/checkout@v3
     - name: Build and test
       run: |
-        dotnet build -c Release -f net8.0
+        dotnet build -c Release
```

Comment:
Under https://github.com/OrchardCMS/OrchardCore/issues/14721#issuecomment-1819346995 aren't you saying that `-f net8.0` is needed?

Reply:
Yes, when multiple TFMs we need to build each TFM after each other but because of the issue this PR tries to fix. Related to translation files to be copied once, not for each TFM while there are inner builds.